### DISCUSSION TITLE
How does `pytest` determine the version of the plugins?

### DISCUSSION THREAD

**Discussion by pmeier** [pytest-dev/pytest#8944] [Q&A] [answered]
If I run `pytest` with my plugin [`pytest-pytorch`](https://github.com/Quansight/pytest-pytorch) I get the following header:

```
=========================================== test session starts ===========================================
platform linux -- Python 3.9.6, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
rootdir: /home/user/pytorch, configfile: pytest.ini
plugins: pytorch-0.0.0, hypothesis-6.14.3
```

As you can see, `pytest` thinks that `pytest-pytorch==0.0.0` is installed. At the same time, the `__version__` attribute seems to be set correctly.

```
$ python -c "import pytest_pytorch; print(pytest_pytorch.__version__)"
0.2.1
```

**Comment by RonnyPfannschmidt** [comment, marked as answer]
The package Metadata is used, so if the dist info is wrong, pytest will display that

**Reply by pmeier** [reply]
Is there an easy command that I can run to get the `0.0.0` version? `importlib.metadata.version(...)`?

**Reply by RonnyPfannschmidt** [reply]
Yes, that one basically
